### ISSUE TITLE
Issue's "sort" dropdown isn't sticky

### ISSUE THREAD

**Issue by mhulse** [isaacs/github#241]
Just sent this:

> Just noticed that the "sort" dropdown isn't "sticky".
> 
> In other words, I want to choose "Recently updated" and then have that
> "stick" even when I close my browser and come back to the issues page
> at a later time/date/session.
> 
> The old issues would "stick" the "sort" option.

**Comment by vanowm** [comment]
It was intention change as a lazy work around of a bug, instead of fixing the bug itself...

**Comment by sampablokuper** [comment]
I completely agree with @vanowm. Also, I note that in their reply to @mhulse, GitHub said:

> we may bring [sticky sorting] back in the future if we can make [it] work better than before.

@mhulse, please can you therefore reopen this issue, as it has not been fixed and there *is* a possibility that it will be fixed at some point in the future.

**Comment by sampablokuper** [comment]
Incidentally, this lack of stickiness is also present (and annoying and inconvenient) at https://github.com/issues .

**Comment by mhulse** [comment]
Thanks for the prod. Re-opened.
